### PR TITLE
[Feature] Lesson schedule

### PR DESCRIPTION
**Pull request by jcostamagna**
## Summary
Add Lesson study to model

## Trello Car
https://trello.com/c/QaZuhz4P/9-modelo-lesson-schedule

### CODE REVIEW

File: app/models/lesson_study.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class LessonStudy < ApplicationRecord
+  belongs_to :course
```
Comment (lmasello) [inline app/models/lesson_study.rb]: All of this should go inside a migration file, which you can generate with:
`rails generate migration createLessonSchedule`

File: app/models/lesson_study.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+class LessonStudy < ApplicationRecord
```
Comment (lmasello) [inline app/models/lesson_study.rb]: It's LessonSchedule

File: app/models/lesson_schedule.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
+class LessonSchedule < ApplicationRecord
+  belongs_to :course
+  belongs_to :classroom
+  validates :type, :day, :hour_start, :hour_end, presence: true
+  validate :validate_start_end
+
+  references :course, index: true, foreign_key: true
```
Comment (lmasello) [inline app/models/lesson_schedule.rb]: These reference lines go only in the migration.

File: db/migrate/20180919032754_create_lesson_schedule.rb [hunk 4]
```diff
@@ -0,0 +1,13 @@
+class CreateLessonSchedule < ActiveRecord::Migration[5.1]
+  def change
+    create_table :lesson_schedules do |t|
```
Comment (lmasello) [inline db/migrate/20180919032754_create_lesson_schedule.rb]: Cool! Then you need to run rails db:migrate. If you are still having pg problems see this repo readme or tell me

File: app/models/lesson_schedule.rb [hunk 5]
```diff
@@ -0,0 +1,15 @@
+class LessonSchedule < ApplicationRecord
+  belongs_to :course
```
Comment (lmasello) [inline app/models/lesson_schedule.rb]: Add this line:
  self.inheritance_column = :_type_disabled # So that we can use the :type column

**Comment by jcostamagna** [comment]
retest this please

**Pull request closed without merging**